cmd/update-tls-ownership/update-tls-ownership.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/openshift/origin/pkg/cmd/generator/certs"
+	"github.com/spf13/cobra"
+)
+
+func main() {
+	root := &cobra.Command{
```

Comment:
Can this command produce a json file https://github.com/openshift/library-go/blob/master/pkg/certs/cert-inspection/certgraphapi/type_registry.go#L4 that is stored in /tls/owner-info?

Reply:
Yes. Updated it to generate `tls/rawTLSInfo/tls-info.json` and `tls/ownership/known-owners.json`